app/router.ts

```diff
@@ -101,37 +101,38 @@ Router.map(function() {
         this.mount('registries');
     }
 
-    /*
-     * Guid Routing
-     *
-     * Root guid URLs (e.g. "/mst3k/") will match the `resolve-guid` route, which
-     * will ask the API what type of object the guid refers to, then transition
-     * to the appropriate `guid-<type>` route below.
-     *
-     * Non-unique nested routes that begin with a guid need special handling.
-     * See `resolve-guid.forks` for an example.
-     */
-    this.route('guid-file', { path: '/:file_guid' });
-    this.route('guid-node', { path: '/:node_guid' }, function() {
+    this.route('guid-file', { path: '--file/:file_guid' });
```

Comment:
Should (can?) we just name all the dynamic segment arguments `guid`, e.g. `'--file/:guid'`? It feels a little redundant now that these are prefixed.

Reply:
Up to you? I don't really have strong feelings one way or another

Reply:
For context, they were all named `:<type>_guid` only because it was necessary to prevent ember router deduping them. You can have multiple routes with the same path *pattern*, but not with the same path *string*. Nice benefit of this approach is we're not depending on that weird, subtle, undocumented behavior, which imo is worse than depending on private API.